rero_ils/modules/locations/api.py

```diff
@@ -129,10 +129,22 @@ def get_links_to_me(self):
 
     def reasons_not_to_delete(self):
         """Get reasons not to delete record."""
+        from ..loans.api import LoansSearch
         cannot_delete = {}
         links = self.get_links_to_me()
         if links:
             cannot_delete['links'] = links
+        other = {}
+        pickup_location_count = LoansSearch() \
+            .filter('term', pickup_location_pid=self.pid).count()
+        if pickup_location_count:
+            other['loans pickup locations'] = pickup_location_count
+        transaction_location_count = LoansSearch() \
+            .filter('term', transaction_location_pid=self.pid).count()
+        if transaction_location_count:
+            other['loans transaction locations'] = transaction_location_count
+        if other:
+            cannot_delete['other'] = other
```

Comment:
Does it say that once a loan has been done for a location, even if the loan is now returned since years, we can't remove the location.
In the real life, if a location accept one day to create a loan, the location could be never removed.
SARCASTIC MODE : it will be easier to tell, that we could never (in any case) delete a location :-)